package.json

```diff
@@ -53,6 +53,9 @@
     "resolve": "^1.20.0",
     "tildify": "^2.0.0"
   },
+  "peerDependencies": {
+    "js-yaml": "^4.1.0"
+  },
```

Comment:
`js-yaml` is only the dependency of `hexo-front-matter` and else. I'd recommend putting `js-yaml` directly to the `dependencies` and letting the package manager do the deduplication.